### PR TITLE
Session middleware err not rails app

### PR DESCRIPTION
**Pull request by sancopanco**
without this getting error with sinatra app

### CODE REVIEW

**Comment by davydovanton** [comment]
@sancopanco could you squash your commits?

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ require 'sidekiq/web'
 require 'sidekiq-statistic'
 
 use Rack::Session::Cookie, secret: 'some unique secret string here'
+Sidekiq::Web.instance_eval { @middleware.reverse! } # Last added, First Run
```
Comment (davydovanton) [inline README.md]: and could you explain why this line is needed?

Comment (sancopanco) [inline README.md]: https://github.com/mperham/sidekiq/issues/2459
this workaround woks but there might be a better solution!

Comment (davydovanton) [inline README.md]: okay
Please, squash your commits and after that I will merge this changes to master

Comment (davydovanton) [inline README.md]: oh, sorry, I did not notice that you have already :sweat:

**Comment by davydovanton** [comment]
Thanks for contribution :+1:

**Pull request merged**